2022-11-03 11:14:40,307 Converting data/part113/in/kerala/2016/2016_059_108.pdf to img...
2022-11-03 11:14:49,639 Parsing first and last page data..
2022-11-03 11:15:06,296 Detecting and parsing data/part113/in/kerala/2016/2016_059_108.pdf boxes..
2022-11-03 11:15:06,296 Processing page 3
2022-11-03 11:15:40,966 Processing page 4
2022-11-03 11:16:15,499 Processing page 5
2022-11-03 11:16:51,931 Processing page 6
2022-11-03 11:17:27,282 Processing page 7
2022-11-03 11:18:02,775 Processing page 8
2022-11-03 11:18:38,637 Processing page 9
2022-11-03 11:19:14,486 Processing page 10
2022-11-03 11:19:50,198 Processing page 11
2022-11-03 11:20:25,564 Processing page 12
2022-11-03 11:20:44,522 Processing page 13
2022-11-03 11:21:21,578 Processing page 14
2022-11-03 11:21:57,930 Processing page 15
2022-11-03 11:22:32,340 Processing page 16
2022-11-03 11:23:06,861 Processing page 17
2022-11-03 11:23:42,277 Processing page 18
2022-11-03 11:24:17,197 Processing page 19
2022-11-03 11:24:52,953 Processing page 20
2022-11-03 11:25:28,866 Processing page 21
2022-11-03 11:26:05,484 Processing page 22
2022-11-03 11:26:42,289 Processing page 23
2022-11-03 11:27:19,359 Processing page 24
2022-11-03 11:27:53,544 Processing page 25
--- Logging error ---
Traceback (most recent call last):
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/logging/__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 39-40: ordinal not in range(256)
Call stack:
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 80, in _worker
    work_item.run()
  File "/home/t-9adar/.conda/envs/env38/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "../parse_unsearchable_rolls/parser/parser.py", line 320, in get_data
    processed_box = self.process_boxes_text(text)
  File "../parse_unsearchable_rolls/parser/parser.py", line 149, in process_boxes_text
    result, last_key, is_splitted = self.columns_split(r, self.columns, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 255, in columns_split
    result, last_key, is_splitted = self.handle_separator_without_column(r, result, last_key)
  File "../parse_unsearchable_rolls/parser/parser.py", line 226, in handle_separator_without_column
    logging.warning(f'Split uncaught {r} {result}')
Message: "Split uncaught \u0d1f\u0d3f WUble>J 2B OrderedDict([('number', '663'), ('id', 'JLC1600709'), ('\u0d2a\u0d47\u0d30\u0d4d', '\u0d38\u0d41\u0d28\u0d3f\u0d24')])"
Arguments: ()
2022-11-03 11:28:11,105 Split uncaught \u0d1f\u0d3f WUble>J 2B OrderedDict([('number', '663'), ('id', 'JLC1600709'), ('\u0d2a\u0d47\u0d30\u0d4d', '\u0d38\u0d41\u0d28\u0d3f\u0d24')])
2022-11-03 11:28:28,322 Processing page 26
2022-11-03 11:29:05,159 Processing page 27
2022-11-03 11:29:43,058 Processing page 28
2022-11-03 11:30:20,213 Processing page 29
2022-11-03 11:30:56,403 Processing page 30
2022-11-03 11:31:31,952 Processing page 31
2022-11-03 11:32:07,564 Processing page 32
2022-11-03 11:32:43,172 Processing page 33
2022-11-03 11:33:19,571 Processing page 34
2022-11-03 11:33:56,961 Processing page 35
2022-11-03 11:34:32,578 Processing page 36
2022-11-03 11:35:09,874 Processing page 37
2022-11-03 11:35:48,023 Processing page 38
2022-11-03 11:36:23,008 Processing page 39
2022-11-03 11:36:57,556 Processing page 40
2022-11-03 11:37:32,178 Processing page 41
2022-11-03 11:38:08,246 Processing page 42
2022-11-03 11:38:43,490 Processing page 43
2022-11-03 11:39:19,608 Processing page 44
2022-11-03 11:39:55,788 Processing page 45
/share/svasudevan2lab/gs-delhi/parse_unsearchable_rolls/scripts/kerala/joblist_2016/kerala113.sh: line 1: 56263 Killed                  python scripts/kerala/kerala_2016.py data/part113/

#################
Epilogue Job Info:
Job ID: 466350[113].acropolis
User ID: t-9adar
Job Name: Kerala2016JobArray-113
Resource List: nodes=1:ppn=1,mem=2gb,neednodes=1:ppn=1
Resources Used: cput=00:24:28,vmem=3428356kb,walltime=00:25:48,mem=2036936kb,energy_used=0
Exit Code: 137
Exit code 137 job terminated, resource limit violation
Generally this means you need to request more RAM or CPU
Please compare your resources used/requested
#################

